docs/drivers.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Driver Requirements
+---
+
+Connectors built with the Connector SDK use either an ODBC or a JDBC driver to communicate with the database.
+
+In general, we prefer JDBC connectors to ODBC connectors, as JDBC is cross-platform.
+
+
+# JDBC Drivers
+To learn more about JDBC drivers and Tableau, check *Specify the right JDBC driver* section in  the  [user documentation](https://help.tableau.com/current/pro/desktop/en-us/examples_otherdatabases_jdbc.htm).
+
+## Requirements
+- You must have read permissions on the JAR file.
+- Tableau requires a JDBC 4.0 or later driver.
+- Tableau requires a Type 4 JDBC driver.
+
+## JDBC Driver Class Isolation
+If the driver only includes a single jar file, copy it to the JDBC driver location. <br/>
+If the driver includes more than a single file, create a unique subfolder under JDBC driver location and include all required files.
+- Windows: C:\Program Files\Tableau\Drivers
+- Mac: ~/Library/Tableau/Drivers
+- Linux: /opt/tableau/tableau_driver/jdbc
+<br/>
+
+ This will create an isolated classloader for that driver.
+
+# ODBC Drivers
+To learn more about ODBC drivers and Tableau, check the *Tableau and ODBC* page in the [user documentation](https://help.tableau.com/current/pro/desktop/en-us/odbc_tableau.htm)
+
+## Requirements
+- Tableau requires an ODBC version of 3 or higher
+
+# Important SQLStates
+
+Tableau expects correct SQLStates to be returned in certain situations. Inaccurate SQLStates (for example, returning a generic error instead of SLQState 28000 for invalid credentials) may cause bugs in the connector.
+
+SQLSTATE | Error | Scenario
```

Comment:
Double check the table renders correctly after publish, unfortunately its a bit hard to predict with gh-pages.